docusaurus/docs/develop/path/cheatsheet.md

```diff
@@ -0,0 +1,240 @@
+---
```

Comment:
Wdyt of adding a gif/video to this page as well?

Reply:
Sure,  I can work on that later today.